coderd/database/queries/prebuilds.sql

```diff
@@ -42,6 +42,11 @@ FROM templates t
 WHERE tvp.desired_instances IS NOT NULL -- Consider only presets that have a prebuild configuration.
 	AND (t.id = sqlc.narg('template_id')::uuid OR sqlc.narg('template_id') IS NULL);
 
+-- name: GetTemplatePresetsByID :one
```

Comment:
- should the name be singular instead of plural?
- move it to `presets.sql`?
- there are similar query: `-- name: GetPresetByID :one`, you may consider reuse

Reply:
Thanks! I missed that.